src/Categories/Morphism/Duality.agda

```diff
@@ -53,3 +55,12 @@ op-≅⇒≅ A≅B = record
   ; iso  = op-Iso⇒Iso iso
   }
   where open Op._≅_ A≅B
+
+
+module DualityConversionProperties where
+  private
+    Iso⇔op-Iso : ∀(iso : Iso f g) → op-Iso⇒Iso (Iso⇒op-Iso iso) ≡ iso
+    Iso⇔op-Iso _ = refl
+
+    ≅⇔op-≅ : ∀(A′ B′ : Obj) → (A′≅B′ : A′ ≅ B′) -> op-≅⇒≅ (≅⇒op-≅ A′≅B′) ≡ A′≅B′
```

Comment:
Since this is all private, this could just be `op-involutive` or, if you want a longer name, `op-≅-involutive`.

Reply:
done